packages/components/src/Chip/Chip.tsx

```diff
@@ -41,19 +42,21 @@ export const Chip = ({
     label,
     heading,
   );
+  const Tag = onClick ? "button" : "div";
 
   return (
-    <Tooltip message={tooltipMessage}>
-      <button
+    <Tooltip message={tooltipMessage} setTabIndex={false}>
```

Comment:
Tooltip always setting the tabIndex to zero which was causing issues with the tabbing/keyboard controls. So I needed a way to selectively disable that functionality in certain scenarios (when used within a Chips)